tests/manage/z_cluster/test_mon_disk_low.py

```diff
@@ -0,0 +1,111 @@
+    def workloads_dir_setup(self, request):
+        """
+        Setting up the environment for the test
+        """
+
+        self.mon_pod = pod.get_mon_pods()[0]
+        self.mon_suffix = self.mon_pod.get().get("metadata").get("labels").get("mon")
+        self.workloads_dir = f"/var/lib/ceph/mon/ceph-{self.mon_suffix}/workloads"
+        log.info(f"Selected mon '{self.mon_pod.name}'")
+        self.mon_pod.exec_cmd_on_pod(f"mkdir {self.workloads_dir}")
+        self.mon_pod.exec_cmd_on_pod(f"touch {self.workloads_dir}/{TEMP_FILE}")
+
+        def finalizer():
+            self.mon_pod.exec_cmd_on_pod(f"rm -rf {self.workloads_dir}")
+            time.sleep(SLEEP_TIMEOUT)
+            utils.ceph_health_check()
+
+        request.addfinalizer(finalizer)
+
+    def get_used_percentage(self):
+        """
+        Get used percentage on /var/lib/ceph/mon/ceph-[a/b/c]
+
+        Returns:
+            int: Used space percentage
+        """
+
+        cmd = f"df -Th | grep /var/lib/ceph/mon/ceph-{self.mon_suffix}"
+        mount_details = self.mon_pod.exec_sh_cmd_on_pod(command=cmd, sh="sh")
+        used_percent = mount_details.split()[5].replace("%", "")
+        return int(used_percent)
+
+    def exec_dd_cmd(self):
+        """
+        Append 1G to tmp file using dd command
+        """
+
+        write_cmd = f"dd if=/dev/urandom of=/var/lib/ceph/mon/ceph-{self.mon_suffix}/workloads/{TEMP_FILE} "
+        write_cmd = write_cmd + f"bs={DD_BLOCK_SIZE}M count={DD_COUNT} "
+        write_cmd = write_cmd + f"seek={self.dd_seek_count * DD_BLOCK_SIZE * DD_COUNT}"
+        self.mon_pod.exec_sh_cmd_on_pod(command=write_cmd, sh="sh")
+        self.dd_seek_count += 1
+
+    @pytest.mark.usefixtures(workloads_dir_setup.__name__)
+    def test_mon_disk_low(self):
+        """
+        Test mon disk threshold
+
+        Steps:
+          - Write to temp file using dd until reaches >= 85% (1G each)
+          - Check ceph health from 80% and above
+          - From 85% and above, ceph health status should be
+            'HEALTH_WARN' with warning message regarding low space
+        """
+
+        used_percent = self.get_used_percentage()
+        log.info(f"Used percentage on {self.workloads_dir}: {used_percent}%")
+
+        should_keep_writing = True
+        while should_keep_writing:
+            self.exec_dd_cmd()
+            used_percent = self.get_used_percentage()
+            log.info(f"Used percentage on {self.workloads_dir}: {used_percent}%")
+            if used_percent >= 80:
+                time.sleep(SLEEP_TIMEOUT)
+                if used_percent >= 85:
+                    ceph_status = CephCluster().get_ceph_health()
+                    log.info(f"Ceph status is: {ceph_status}")
+                    assert (
+                        "HEALTH_WARN" in ceph_status
+                        and "low on available space" in ceph_status
+                    ), "Ceph status should be HEALTH_WARN containing 'low on available space'"
```

Comment:
We can use here the function `run_cmd_verify_cli_output` in the `helpers` file: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py#L3188. 

> assert run_cmd_verify_cli_output(
        cmd="ceph health detail",
        expected_output_lst={"HEALTH_WARN", "low on available space"},
        cephtool_cmd=True,
    ), "Ceph status should be HEALTH_WARN containing 'low on available space'"

Reply:
done